cmd/tailscale/cli/configure-kube.go

```diff
@@ -39,6 +41,19 @@ The hostname argument should be set to the Tailscale hostname of the peer runnin
 	Exec: runConfigureKubeconfig,
 }
 
+// kubeconfigBaseDir returns the base directory where the .kube directory
+// should exist.
+func kubeconfigBaseDir() string {
+	if runtime.GOOS == "windows" {
+		homedir.HomeDir()
+	}
+	h := os.Getenv("HOME")
```

Comment:
FWIW `homedir.HomeDir()` also returns this on non-Windows, so you could keep using that value, to minimize the divergence between the Windows and non-Windows code paths.

Reply:
Redid this a bit to return the full path, PTAL